cli/src/commands/content/updateChannelPayoutsProposal.ts

```diff
@@ -0,0 +1,122 @@
+  static description = 'Create channel payouts proposal.'
+  static flags = {
+    title: flags.string({
+      char: 't',
+      required: true,
+      description: 'Title of the proposal',
+    }),
+    description: flags.string({
+      char: 'd',
+      required: true,
+      description: 'Description of the proposal',
+    }),
+    exactExecutionBlock: flags.integer({
+      char: 'b',
+      required: false,
+      description: 'The Block at which the proposal should be automatically executed',
+    }),
+    stakingAccountId: flags.string({
+      char: 's',
+      required: true,
+      description: 'Proposer staking account Id',
+    }),
+    min: flags.integer({
+      required: false,
+      description: 'Minimum cashout amount allowad to a channel',
+    }),
+    max: flags.integer({
+      required: false,
+      description: 'Maximum cashout amount allowed to a channel',
+    }),
+    channelCashoutsEnabled: flags.boolean({
+      char: 'e',
+      required: false,
+      description: 'Whether cashouts be enabled/disabled',
+    }),
+    payloadFilePath: flags.string({
+      char: 'p',
+      required: false,
+      description: 'Path to protobuf serialized file containing channel payouts payload',
+    }),
+    ...UploadCommandBase.flags,
+  }
+
+  async run(): Promise<void> {
+    const {
+      title,
+      description,
+      exactExecutionBlock,
+      stakingAccountId,
+      min,
+      max,
+      channelCashoutsEnabled,
+      payloadFilePath,
+    } = this.parse(UpdateChannelPayoutsProposal).flags
+
+    // Context
+    const member = await this.getRequiredMemberContext()
+    const keypair = await this.getDecodedPair(member.membership.controllerAccount)
+
+    const expectedDataSizeFee = await this.getApi().dataObjectPerMegabyteFee()
+    const expectedDataObjectStateBloatBond = await this.getApi().dataObjectStateBloatBond()
+    const minCashoutAllowed = min || (await this.getOriginalApi().query.content.minCashoutAllowed())
+    const maxCashoutAllowed = max || (await this.getOriginalApi().query.content.maxCashoutAllowed())
+
+    this.jsonPrettyPrint(
+      JSON.stringify({
+        title,
+        description,
+        exactExecutionBlock,
+        stakingAccountId,
+        minCashoutAllowed,
+        maxCashoutAllowed,
+        channelCashoutsEnabled,
+      })
+    )
+
+    await this.requireConfirmation('Do you confirm the provided input?', true)
+
+    const result = await this.sendAndFollowNamedTx(keypair, 'proposalsCodex', 'createProposal', [
+      {
+        memberId: member.id,
+        title,
+        description,
+        exactExecutionBlock,
+        stakingAccountId,
+      },
+      {
+        UpdateChannelPayouts: createType('PalletContentUpdateChannelPayoutsParametersRecord', {
+          commitment: payloadFilePath ? await generateCommitmentFromPayloadFile('PATH', payloadFilePath) : null,
+          payload: payloadFilePath
+            ? {
+                uploaderAccount: keypair.address,
```

Comment:
The runtime has been updated, the uploaderAccount is no longer passed in the payload. It will be the account of the proposal. See https://github.com/Joystream/joystream/pull/4570

Reply:
Addressed in [7dfea16](https://github.com/Joystream/joystream/pull/4441/commits/7dfea165b390d0bf30dcfe5da5110a6043b931c4)